hrjobcontract/CRM/Hrjobcontract/DAO/HRJobContract.php

```diff
@@ -151,6 +151,8 @@ static function &fields()
           'export' => false,
           'import' => false,
           'where' => 'civicrm_hrjobcontract.id',
+          'entity' => 'HRJobContract',
+          'bao' => 'CRM_Hrjobcontract_DAO_HRJobContract',
```

Comment:
Some of the DAOs have an extra `localizable` field, but other don't. What is this based on?

Reply:
It's used only for multi-language sites. It comes from the schema xml files. If it's not present in these files, GenCode will automatically set them to 0. I've manually did this for the Job Roles and Contracts DAOs